### PR TITLE
feat: add additional verbose output to js_binary launcher to indicate cwd

### PR DESCRIPTION
**Pull request by gregmagolan**

### CODE REVIEW

File: js/private/js_binary.sh.tpl [hunk 1]
```diff
@@ -129,6 +129,9 @@ else
         printf "\nERROR: %s: BAZEL_BINDIR must be set in environment when not running out of runfiles so js_binary can run out of the output tree on build actions\n" "$LOG_PREFIX" >&2
         exit 1
     fi
+    if [ "${JS_BINARY__VERBOSE:-}" ]; then
```
Comment (f0rmiga) [inline js/private/js_binary.sh.tpl:132]:
```suggestion
    if [ -n "${JS_BINARY__VERBOSE:-}" ]; then
```

Comment (f0rmiga) [reply js/private/js_binary.sh.tpl:132]: I'm surprised it works without `-n`!

**Review by f0rmiga — APPROVED** [review]

File: js/private/js_binary.sh.tpl (hunk 1 above)
Comment (gregmagolan) [reply js/private/js_binary.sh.tpl:132]: `-n` is the implicit default :) 

```
[ -n STRING ] or [ STRING ] | True if the length of "STRING" is non-zero.
```

https://tldp.org/LDP/Bash-Beginners-Guide/html/sect_07_01.html#sect_07_01_02

Comment (f0rmiga) [reply js/private/js_binary.sh.tpl:132]: TIL

**Pull request merged**